src/GraphQL/Types/IAbstractGraphType.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GraphQL.Types
+{
+    public interface IAbstractGraphType
```

Comment:
swiched to an interface mostly to manage InterfaceGraphType being both a ComplexType and an Abstract one